libs/designsystem/slide/src/slides.component.html

```diff
@@ -1,5 +1,9 @@
 <div class="navigation" *ngIf="showNavigation || title !== undefined">
-  <span class="kirby-text-medium">{{ title }}</span>
+  <span class="kirby-text-medium">
+    <ng-content select="h1,h2,h3,h4,h5,h6">{{ title }}</ng-content>
```

Comment:
Headings have a margin that we should make sure to remove, otherwise it will be mis-aligned like in the example:

<img width="730" alt="billede" src="https://github.com/user-attachments/assets/a25b2540-66f7-4698-9114-8b16cb61cb82" />